nexus_constructor/model/module.py

```diff
@@ -97,20 +97,26 @@ def as_dict(self, error_collector: List[str]):
 
 
 @attr.s
-class F142Stream(EV42Stream):
+class F142Stream(StreamModule):
+    cue_interval = attr.ib(type=int, default=None, init=False)
```

Comment:
`init=False ` is not required probably for `cue_interval` and `chunk_size`

Reply:
True! Done